packages/kbn-alerts-ui-shared/src/rule_form/rule_actions/rule_actions_settings.tsx

```diff
@@ -0,0 +1,263 @@
+    }
+  }, [minimumActionThrottle, minimumActionThrottleUnit, actionThrottle, actionThrottleUnit]);
+
+  const showActionAlertsFilter =
+    hasFieldsForAad({
+      ruleType: selectedRuleType,
+      consumer,
+      validConsumers,
+    }) || producerId === AlertConsumers.SIEM;
+
+  return (
+    <EuiFlexGroup direction="column" data-test-subj="ruleActionsSettings">
+      <EuiFlexItem>
+        <EuiFlexGroup>
+          <EuiFlexItem>
+            <RuleActionsNotifyWhen
+              frequency={action.frequency}
+              throttle={actionThrottle}
+              throttleUnit={actionThrottleUnit}
+              hasAlertsMappings={selectedRuleType.hasAlertsMappings}
+              onChange={onNotifyWhenChange}
+              onUseDefaultMessage={onUseDefaultMessageChange}
+              showMinimumThrottleWarning={showMinimumThrottleWarning}
+              showMinimumThrottleUnitWarning={showMinimumThrottleUnitWarning}
+            />
+          </EuiFlexItem>
+          <EuiFlexItem>
+            {showSelectActionGroup && (
+              <EuiSuperSelect
+                prepend={
+                  <EuiFormLabel htmlFor={`addNewActionConnectorActionGroup-${action.actionTypeId}`}>
+                    {ACTION_GROUP_RUN_WHEN}
+                  </EuiFormLabel>
+                }
+                data-test-subj="ruleActionsSettingsSelectActionGroup"
+                fullWidth
+                id={`addNewActionConnectorActionGroup-${action.actionTypeId}`}
+                options={actionGroups.map(({ id: value, name }) => ({
+                  value,
+                  ['data-test-subj']: `addNewActionConnectorActionGroup-${value}`,
+                  inputDisplay: actionGroupDisplay({
+                    ruleType: selectedRuleType,
+                    actionGroupId: value,
+                    actionGroupName: name,
+                    actionTypeId: action.actionTypeId,
+                  }),
+                  disabled: isActionGroupDisabledForActionType(
+                    selectedRuleType,
+                    value,
+                    action.actionTypeId
+                  ),
+                }))}
+                valueOfSelected={selectedActionGroup.id}
+                onChange={onActionGroupChange}
+              />
+            )}
+          </EuiFlexItem>
+        </EuiFlexGroup>
+      </EuiFlexItem>
+      {showActionAlertsFilter && (
+        <EuiFlexItem>
+          <EuiFlexGroup direction="column">
+            <EuiFlexItem>
+              <EuiFormRow
+                fullWidth
+                error={actionError.filterQuery}
+                isInvalid={!!actionError.filterQuery?.length}
+              >
+                <RuleActionsAlertsFilter
+                  action={action}
+                  onChange={onAlertsFilterChange}
+                  featureIds={[
+                    producerId === 'alerts' ? 'stackAlerts' : (producerId as ValidFeatureId),
+                  ]}
+                  appName="stackAlerts"
```

Comment:
Is it possible for this component to be used outside the stack page?

Reply:
Yes, I (or someone) will make these dynamic when we do so